ignite/utils.py

```diff
@@ -285,7 +285,10 @@ def wrapper(*args: Any, **kwargs: Dict[str, Any]) -> Callable:
     return decorator
 
 
-def hash_checkpoint(checkpoint_path: Union[str, Path], output_dir: Union[str, Path],) -> Tuple[Path, str]:
+def hash_checkpoint(
+    checkpoint_path: Union[str, Path],
+    output_dir: Union[str, Path],
```

Comment:
```suggestion
    output_dir: Union[str, Path]
```
@louis-she, please remove this comma and rerun black